websites/L/Lazada/metadata.json

```
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "392970361047613441",
+		"name": "niqowi"
+	},
+	"service": "Lazada",
+	"description": {
+		"en": "Visit Lazada.com to get incredible bargains and stacking coupons for even more savings."
+	},
+	"url": "lazada.com",
```

Site supports `www` subdomain
```suggestion
	"url": [
		"lazada.com",
		"www.lazada.com"
	],
```